hackathon_site/registration/jinja2/registration/application.html

```diff
@@ -31,19 +31,50 @@
 {% endblock %}
 
 {% block form_inputs %}
-    {% for field in form %}
-        {% if field.field.widget.__class__.__name__ == "Textarea" %}
-            {{ render_field(field, size="s12", show_help_text_as_label=True) }}
-        {% elif field.field.widget.input_type == "select" %}
-            {{ render_field(field, size="s12 select2-wrapper") }}
-        {% elif field.field.widget.input_type == "file" %}
-            {{ render_field(field, size="s12", show_label=False) }}
-        {% elif field.field.widget.input_type == "checkbox" %}
-            {{ render_checkbox_field(field, size="s12") }}
-        {% else %}
-            {{ render_field(field, size="s12") }}
-        {% endif %}
-    {% endfor %}
+    <div class="col s12">
+        <h4>Personal Information</h4>
+        <br />
+    </div>
+    {{ render_field(form.birthday, size="s12 m6") }}
+    {{ render_field(form.phone_number, size="s12 m6") }}
+    {{ render_field(form.gender, size="s12 m6 select2-wrapper") }}
+    {{ render_field(form.ethnicity, size="s12 m6 select2-wrapper") }}
+    {{ render_field(form.tshirt_size, size="s12 m6 select2-wrapper") }}
+
+    <div class="col s12">
+        <h4>Address Information</h4>
+        <br />
+    </div>
+    {{ render_field(form.address1, size="s12 m6", show_help_text=True) }}
+    {{ render_field(form.address2, size="s12 m6", show_help_text=True) }}
+    {{ render_field(form.city, size="s12 m6") }}
+    {{ render_field(form.region, size="s12 m6") }}
+    {{ render_field(form.country, size="s12 m6") }}
+    {{ render_field(form.postal_code, size="s12 m6") }}
+
+    <div class="col s12">
+        <h4>Education</h4>
+    </div>
+    {{ render_field(form.school, size="s12 m6 select2-wrapper") }}
+    {{ render_field(form.study_level, size="s12 m6 select2-wrapper") }}
+    {{ render_field(form.graduation_year, size="s12 m6 select2-wrapper") }}
+    {{ render_field(form.program, size="s12 m6 select2-wrapper", show_help_text_as_label=True) }}
+    {{ render_field(form.resume, size="s12 marginTop0 select2-wrapper", show_label=False) }}
+
+    <div class="col s12">
+        <h4>Questions</h4>
+    </div>
+    {{ render_field(form.how_many_hackathons, show_help_text_as_label=True, size="s12 select2-wrapper") }}
+    {{ render_field(form.what_hackathon_experience, show_help_text_as_label=True) }}
+    {{ render_field(form.why_participate, show_help_text_as_label=True) }}
+    {{ render_field(form.what_technical_experience, show_help_text_as_label=True) }}
+    {{ render_field(form.referral_source, show_help_text_as_label=True) }}
```

Comment:
For these 4 long answers, you'll have to update the field ids in `hackathon_site/event/static/event/js/application.js` so the inputs will show a counter of the characters:
<img width="833" alt="Screen Shot 2021-12-22 at 5 09 36 PM" src="https://user-images.githubusercontent.com/42653157/147160770-c67e5faf-e4cd-4455-8eba-a92242ff5bc3.png">

In `/application.js`, just change `"#id_q1, #id_q2, #id_q3"` to `"#id_what_hackathon_experience, #id_why_participate, etc...."`